### PR TITLE
Update contributors.md

### PR DESCRIPTION
**Pull request by cnharris10**
Add additional sections for Steering Committee and Maintainers.

### CODE REVIEW

File: specification/contributors.md
Comment (udam-f2) [inline specification/contributors.md:32]:
```suggestion
* Christopher Harris (Datadog)
* Dennis Park-Rodriguez (Amazon Web Services)
* Erik Peterzon (CloudZero)
* Irena Jurica (Neos)
* Joaquin Prado (FinOps Foundation)
* Karl Kraft (Walmart)
* Larry Advey (Twilio)
* Michael Flanakin (Microsoft)
* Mike Fuller (FinOps Foundation)
* Riley Jenkins (DOMO)
* Rupa Patel (Google)
* Udam Dewaraja (FinOps Foundation)
```

File: specification/contributors.md
Comment (udam-f2) [inline specification/contributors.md:11]:
```suggestion
```

File: specification/contributors.md
Comment (udam-f2) [inline specification/contributors.md:1]: Suggested order:
## Working Group
### Maintainers
...

### Members
...

## Steering Committee
...

File: specification/contributors.md
Comment (udam-f2) [inline specification/contributors.md:33]: Seems like we would want to at least not double count members between maintainers and members

Comment (cnharris10) [reply specification/contributors.md:33]: Was thinking the same but don't know a good way to manifest this other than 1 list like:

```
John Doe
Jane Doe [Maintainer]
Mike Doe [Steering Committee]
Alex Doe [Maintainer, Steering Committee]
```

Comment (cnharris10) [reply specification/contributors.md:23]: Will just remove Maintainers from Member section for now.

**Review by SonalGarg-SG — APPROVED** [review]

File: specification/contributors.md
Comment (kk09v) [inline specification/contributors.md:10]: Correcting Erik's name
```suggestion
* Erik Peterson (CloudZero)
```

**Review by flanakin — APPROVED** [review]

**Comment by jpradocueva** [comment]
Members are requested to review the following spreadsheet [FOCUS Attendance & Contributions](https://docs.google.com/spreadsheets/d/1D9RDPNuf_5E-yk8OOeUizHjj_HrZwhExkP5sP4iuZ28/edit?usp=sharing) and add their names to the column `Release v1.0 Contributor` with a "YES" if they want their name to be listed in the contribution section of the FOCUS Realse v1.0 specifications.

**Review by rileyjenk — DISMISSED** [review]

File: specification/contributors.md
Comment (rileyjenk) [reply specification/contributors.md:17]: Marketing reminded me it isn't DOMO...

**Review by ljadvey — APPROVED** [review]

File: specification/contributors.md
Comment (flanakin) [inline specification/contributors.md:18]: Sorry, Tim! 😶‍🌫️
```suggestion
```

File: specification/contributors.md
Comment (flanakin) [inline specification/contributors.md:31]: Bring Anne back
```suggestion
* Anne Johnston (Capital One)
* Arun Ramakrishnan (Oracle)
```

File: specification/contributors.md
Comment (flanakin) [inline specification/contributors.md:73]:
```suggestion
* Tim O'Brien (Walmart)
* Trey Morgan (Microsoft)
```

**Review by flanakin — APPROVED** [review]

File: specification/contributors.md
Comment (udam-f2) [inline specification/contributors.md:5]:
```suggestion
Thanks to the following FOCUS Maintainers for their leadership and contributions to the FOCUS specification.
```

File: specification/contributors.md
Comment (udam-f2) [inline specification/contributors.md:82]:
```suggestion
Thanks to the following FOCUS Steering Committee members for their leadership on the FOCUS specification.
```

File: specification/contributors.md
Comment (udam-f2) [inline specification/contributors.md:16]:
```suggestion
* Riley Jenkins (Domo)
```

File: specification/contributors.md
Comment (udam-f2) [inline specification/contributors.md:41]:
```suggestion
* George Parker (Salesforce)
* Graham Murphy (Australian Retirement Trust)
```

**Review by projectlocale — COMMENTED** [review]
Overall, the idea of different levels of contributors is good. I have a few concerns though. 

First, the distinction is very vague. There are some people in the contributors list that I don't remember speaking in meetings or even attending and some people who were regulars. Maybe I just lost track of things, but I wonder where the clarifying definitions of the distinctions for this version are. 

I think that metaphorically they should be stapled to the categories so it's really clear what the current working version of those terms mean. Other than clearly qualifying the definitions right next to its use I have no concerns. There is no one who I would add or delete from the list because this spec and what it takes to count as a contributor has been developed in an ad hoc and somewhat retrospective manner. That's a respectable level of process maturity at this point in the development of the group, I just think that it needs to be qualified.

**Comment by udam-f2** [comment]
> Overall, the idea of different levels of contributors is good. I have a few concerns though.
> 
> First, the distinction is very vague. There are some people in the contributors list that I don't remember speaking in meetings or even attending and some people who were regulars. Maybe I just lost track of things, but I wonder where the clarifying definitions of the distinctions for this version are.
> 
> I think that metaphorically they should be stapled to the categories so it's really clear what the current working version of those terms mean. Other than clearly qualifying the definitions right next to its use I have no concerns. There is no one who I would add or delete from the list because this spec and what it takes to count as a contributor has been developed in an ad hoc and somewhat retrospective manner. That's a respectable level of process maturity at this point in the development of the group, I just think that it needs to be qualified.

The definition is here: https://docs.google.com/document/d/13KWYKjM09FwXkm9Z-eOHafEBspYI277szZqA4WyaoA0/edit#heading=h.hn2ldkkpe4b8. It was created as a part of the last round to avoid some of the ambiguity (though it's not possible to have a 100% accurate filter). We should try to get it into the git so its more widely available.

Specifically, joining meetings but not driving discussions, issues, authoring/co-authoring, Reviewing PRs would not qualify as a contributor per the requirements the group agreed to. 

Since you only joined in the last round, you may not recognize many names that were v0.5 or v1.0-preview contributors that haven't been recently active.

**Comment by projectlocale** [comment]
> > Overall, the idea of different levels of contributors is good. I have a few concerns though.
> > First, the distinction is very vague. There are some people in the contributors list that I don't remember speaking in meetings or even attending and some people who were regulars. Maybe I just lost track of things, but I wonder where the clarifying definitions of the distinctions for this version are.
> > I think that metaphorically they should be stapled to the categories so it's really clear what the current working version of those terms mean. Other than clearly qualifying the definitions right next to its use I have no concerns. There is no one who I would add or delete from the list because this spec and what it takes to count as a contributor has been developed in an ad hoc and somewhat retrospective manner. That's a respectable level of process maturity at this point in the development of the group, I just think that it needs to be qualified.

I agree that it should be put up for merging. This document was created in August of last year before I fully joined, but doesn't specifically mention which round it will become active for among other things. At the time I hadn't figured out enough about how the different procedures for FOCUS worked, but I will definitely have some things to address when this gets added to a pull request. I think this is different from the nature of the discussion for the 1.0 release in the past few weeks, so I think that would need to be reconciled before it is merged. I'm guessing in the interest of time (we're more than 2 weeks behind) and recent discussions no contributor standards will be merged until 1.1.
> The definition is here: https://docs.google.com/document/d/13KWYKjM09FwXkm9Z-eOHafEBspYI277szZqA4WyaoA0/edit#heading=h.hn2ldkkpe4b8. It was created as a part of the last round to avoid some of the ambiguity (though it's not possible to have a 100% accurate filter). We should try to get it into the git so its more widely available.

This is a fair point here:
> Since you only joined in the last round, you may not recognize many names that were v0.5 or v1.0-preview contributors that haven't been recently active.
Not considering the level of contributions of people in earlier rounds was an assumption on my part. I was using the language about grandfathering in contributions here and my memory of recent attendance, which obviously leaves a hole. To be clear, I wasn't contesting anyone's qualifications for this round. Though I can see how talking about the issues with the definition and where it is disclosed could be misconstrued as a judgement of individual members (not my intent). 

Since I see that Venu and I were added  as contributors in this PR (which is correct), and the standards discussion was going to be pushed until 1.1 when everyone has more time, I'll vote for the specific edit and we can revisit the existing questions around contributor definitions specific to future releases.

**Review by projectlocale — APPROVED** [review]
List looks good as proposed. As I've said there are some definition concerns, but I think we can work those out in 1.1.

**Review by projectlocale — APPROVED** [review]
Nothing to add that wasn't already in the Conversation.

**Comment by jpradocueva** [comment]
The Maintainers group agreed this PR on May 13.

**Pull request merged**
